cves/kernel/CVE-2020-2732.yml

```diff
@@ -472,7 +497,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    Yes, because originally the default check of intercepts in the KVM did not
+    actually check the intercepts. The check defaulted to continue no matter 
+    what the output was. This looked like unfinished code.
```

Comment:
Do you think there were any mistakes with the testing or design process? I would also include them here if you can think of any.

Reply:
I don't think that there were any mistakes with the testing process, and upon further research it looks like multiple intercepts were added after this was written. In my honest opinion, this was a normal mistake of not updating old code.